python/templates/datamodel.h.jinja2

```diff
@@ -13,13 +13,25 @@ namespace {{ package_name }} {
 
 using {{ package_name }}DataTypes = podio::utils::TypeList<
 {% for name in datatypes %}
-  {{ name }}{% if not loop.last %},{% endif %}
+{% if loop.first %}  {% else %} {% endif %}{{ name }}{% if not loop.last %},{% endif %}
+{% endfor %}
+>;
```

Comment:
This looks suspiciously like it could simply be a 
```jinja2
using {{ package_name }}DataTypes = podio::utils::TypeList<
  {{ datatypes | map(attribute="name") | join(", ") }}
>;
```

Similar below.

For the version with the collections, it might be necessary to implement a small filter, similar to `quoted_sv` that we already have:
https://github.com/AIDASoft/podio/blob/73f77abe6739aad130d8daf30cdf4a455756100a/python/podio_gen/cpp_generator.py#L609-L612
usage:
https://github.com/AIDASoft/podio/blob/73f77abe6739aad130d8daf30cdf4a455756100a/python/templates/DatamodelDefinition.h.jinja2#L24

Reply:
Simplified but without filter, adding `Collection` to `join` and making sure it's added to the last element.